spec/lib/pdf_filler/nj1040_pdf_spec.rb

```diff
@@ -1739,5 +1739,50 @@
         expect(tax_credit).to eq 600
       end
     end
+
+    describe "gubernatorial elections fund" do
+      context "not mfj" do 
+        let(:intake) { 
+          create(:state_file_nj_intake)
+        }
+        it "does not fill the field when the answer is no" do 
```

Comment:
[pebble] `does not fill the field when the answer is no` is for XML but for PDF `marks no when the answer is no`